web/src/views/ConsentPortal/OpenIDConnectConsentPortal/OpenIDConnectConsentDecisionFormView.tsx

```
@@ -165,128 +167,143 @@
-                                                        checked={claimChecked(claim)}
-                                                        onChange={handleClaimCheckboxOnChange}
+                                </Box>
+                            </Grid>
+                            <Grid size={{ xs: 12 }}>
+                                <Box>{translate("The above application is requesting the following permissions")}:</Box>
+                            </Grid>
+                            <Grid size={{ xs: 12 }}>
+                                <Box className={classes.scopesListContainer}>
+                                    <List className={classes.scopesList}>
+                                        {response?.scopes.map((scope: string) => (
+                                            <Tooltip title={translate("Scope", { name: scope, ns: "consent" })}>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)